akka-actor-testkit-typed/src/main/mima-filters/2.6.10.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# changes to package private and internal implementation classes (#29903)
+
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.actor.testkit.typed.internal.StubbedActorContext.timer")
```

Comment:
This should not be a file but a directory called `2.6.10.backwards.excludes` with a file for the excludes just like the akka-actor-typed mima filters.